pkg/scheduler/framework/session_plugins.go

```diff
@@ -633,13 +634,14 @@ func (ssn *Session) TaskOrderFn(l, r interface{}) bool {
 		return res < 0
 	}
 
-	// If no task order funcs, order task by CreationTimestamp first, then by UID.
+	// If no task order funcs, order task by index first, then by CreationTimestamp.
 	lv := l.(*api.TaskInfo)
 	rv := r.(*api.TaskInfo)
-	if lv.Pod.CreationTimestamp.Equal(&rv.Pod.CreationTimestamp) {
-		return lv.UID < rv.UID
+	res := helpers.ComparePodByIndex(lv.Pod, rv.Pod)
```

Comment:
Prefer to `helpers.CompareTasksByIndex(lv, rv)`; and maybe we can just use `helpers.CompareTasks(lv, rv)` which also incloudes timestamp comparation.